### DISCUSSION TITLE
Hand gestures gameplay

### DISCUSSION THREAD

**Discussion by StellaTham** [Okuu-Sama/Tama-Project-2021#10] [Ideas]
After reading about hand tracking best practices: https://developer.oculus.com/learn/hands-design-bp/ I think that we should think more about our gameplay with hands, especially regarding user experience and comfort. If I understood well what we decided at the beginning, it seems that our player would have to keep their hands in front of the tracks and interact with the hands in the same space during the whole song. Maybe it is too much restrictive? I think that the player would be more at ease if they move their hands more freely. I don't know much for now about hand tracking precisely but according to the linked article, hand tracking is best when it's used to do things that you cannot normally do with controllers. And what we designed for now seems to be possible with controllers so maybe we could think of another way to improve the gameplay? I saw for example somebody tracking what the hands do with the fingers (like detecting victory sign or thumbs up). We could take these into account in our gameplay design?

It is just a small thought I got when doing research of course I don't absolutely want to change what we decided before. I just want us to think a bit more about user experience with hand tracking, and take new possibilities into account! 👍

**Comment by Okuu-Sama** [comment]
Yes we could adapt the gameplay to make it more confortable with handtracking. Maybe the handtracking could be oriented toward interactions with NPCs or monsters after finishing a song. (And not base the whole gameplay of a song on the handtracking as it could be easily tiring).
